src/WarmUps/ArrayListMethods.java

```diff
@@ -0,0 +1,26 @@
+package WarmUps;
+
+import java.util.ArrayList;
+
+public class ArrayListMethods {
+	public static void main(String[] args) {
+		ArrayList<Integer> nums = new ArrayList<>();
+		nums.add(2);
+		System.out.println(nums.contains(3));
+		System.out.println(nums.contains(2));
+	
+		ArrayList<String> fruits = new ArrayList<>();
+		fruits.add("Strawberry");
+		fruits.add("Kiwi");
+		System.out.println(fruits.indexOf("Kiwi"));
+		System.out.println(fruits.indexOf("apple"));
```

Comment:
use the same capitalization for your strings, either have apple start with an uppercase letter or have Strawberry and Kiwi begin with a lowercase letter